util/command.js

```diff
@@ -63,15 +66,15 @@ var init = function () {
 /**
  * Default delimiter used to tokenize message is space(' ') character. To handle quoted strings, which even with spaces,
  * should be considered as a single entity, spaces inside quoted strings are replaced by a space replacement pattern.
- * The default space replacement pattern is {SPACE}, i.e. a string in message like "Firstname Lastname", will be 
+ * The default space replacement pattern is {SPACE}, i.e. a string in message like "Firstname Lastname", will be
  * given as Firstname{SPACE}Lastname. The app handling these tokens should replace this pattern back to space(' ')
  * character. Also, continuous multiple spaces in the message will be replaced by a single space character.
- * 
+ *
  * @param  {String} - Text/message entered by a user in slack channel
  * @return {Object} - Object containing user, channel, command and arguments, all parsed from the user message
  */
 var parse = function (message) {
-	var cleanMessage 	= message.text && message.text.trim().replace(util.regex.regex.multi_spaces, util.config.get('COMMAND_DELIM')),
+	var cleanMessage 	= message.text && message.text.trim().replace(util.regex.regex.multi_spaces, util.config.get('COMMAND_DELIM')) || '',
```

Comment:
Other changes look good. Thanks for the improvements. Please remove this check, and I can merge it.